test/unit/node/heart.test.ts

```diff
@@ -33,17 +33,26 @@ describe("Heart", () => {
     const pathToFile = `${testDir}/file.txt`
     await writeFile(pathToFile, text)
     const fileContents = await readFile(pathToFile, { encoding: "utf8" })
-    const fileStatusBeforeEdit = await stat(pathToFile)
+    // Explicitly set the modified time to 0 so that we can check
+    // that the file was indeed modified after calling heart.beat().
+    // This works around any potential race conditions.
+    const fileHandle = await open(pathToFile, "r+")
+    await fileHandle.utimes(0, 0)
```

Comment:
Whoops, sorry I linked you to the wrong section that makes you open a file handle.  I think you might have to close this as well?  But it might be easier to this one: https://nodejs.org/api/fs.html#fspromisesutimespath-atime-mtime

Reply:
Oh, no, that's my bad. I assumed you could only access `utimes` after opening the file and didn't bother checking the rest of the docs. Yeah, that would leak since I don't have logic to close it (I've run into that with Deno).

Nice!!! Our codebase thanks you for your attention-to-detail 😂